### PR TITLE
Remove 'experimental' from description in iris/fileformats/um/_fast_load_structured_fields.py

### PR DESCRIPTION
**Pull request by djkirkham**
This code is no longer experimental so this comment should be updated.

### CODE REVIEW

**Comment by rhattersley** [comment]
:+1: 

There's another occurrence of `experimental` in a docstring near the end of the file - would you be OK to update that one too?

**Comment by djkirkham** [comment]
Urgh... now the line is too long...

**Comment by djkirkham** [comment]
@rhattersley Can I do 
`:class:`
`` `~iris.fileformats.um._fast_load_structured_fields.FieldCollation` ``
to get it to fit within 80 characters?

**Comment by ajdawson** [comment]
Can we set a default role for iris documentation of `:py:obj:` or even `:any:` so we don't have to specify `:class:`, `:meth:`, `:function:` etc.? Sphinx has supported this for a really long time and it makes documentation easier to write, and easier to read.

**Comment by ajdawson** [comment]
See:
* http://www.sphinx-doc.org/en/stable/config.html#confval-default_role
* http://www.sphinx-doc.org/en/stable/markup/inline.html#inline-markup

**Comment by dkillick** [comment]
> Can we set a default role for iris documentation

@ajdawson that's a great idea in principle but perhaps this PR in particular isn't the best place to try and introduce it? It could make this small PR to fix one specific documentation oversight into a much larger piece of work...

**Comment by ajdawson** [comment]
I'm not suggesting it gets changed here. Just that it gets changed to make this kind of small fix unnecessary (or occur less often) in future.

**Comment by dkillick** [comment]
> Just that it gets changed

:+1:

File: lib/iris/fileformats/um/_fast_load_structured_fields.py [hunk 1]
```diff
@@ -240,7 +240,7 @@ def group_structured_fields(field_iterator):
 
     Returns:
         An generator of
```
Comment (rhattersley) [inline lib/iris/fileformats/um/_fast_load_structured_fields.py]: An -> A

**Comment by rhattersley** [comment]
> @rhattersley Can I do
> :class:
> `~iris.fileformats.um._fast_load_structured_fields.FieldCollation`
> to get it to fit within 80 characters?

It's in a private module so it won't end up in our Sphinx documentation. So how about ditching the Sphinx markup and just saying `A generator of FieldCollation objects...`

**Comment by rhattersley** [comment]
Thanks @djkirkham :+1:

**Pull request merged**